.idea/codeStyles/codeStyleConfig.xml

```diff
@@ -0,0 +1,5 @@
+<component name="ProjectCodeStyleConfiguration">
```

Comment:
Be sure to update your `.gitignore` so that these files don't keep getting updated/pushed to GitHub